src/components/common/Link/StyledLink.js

```diff
@@ -0,0 +1,23 @@
+
+import { NavLink } from 'react-router-dom'
+import styled from 'styled-components'
+import Theme from '../Theme'
+
+const { text, secondary_text, light_primary } = Theme
+const StyledLink = styled(NavLink)`
```

Comment:
Nice 👍 , probably only missed the `.active` class style which differentiates `Link` from `NavLink`.